### ISSUE TITLE
Please, add bulma dependence from the NPM.

### ISSUE THREAD

**Issue by alexsandro-xpt** [cssninjaStudio/fresh#3]
Adding bulma dependency from package.json is more smarter.

**Comment by cssninjaStudio** [comment]
Yes it is planned, this starter is going to be turned into a gulp starter with npm package management. Bulma will be upgraded to 0.7. Should come live very soon.
